lbrynet/core/StreamDescriptor.py

```diff
@@ -263,7 +264,7 @@ def save_sd_info(blob_manager, sd_hash, sd_info):
                                                (sd_hash, calculated_sd_hash))
     stream_hash = yield blob_manager.storage.get_stream_hash_for_sd_hash(sd_hash)
     if not stream_hash:
-        log.debug("Saving info for %s", sd_info['stream_name'].decode('hex'))
+        log.debug("Saving info for %s", binascii.unhexlify(sd_info['stream_name']))
```

Comment:
I would do `from binascii import unhexlify`, it's used often enough that it's more convenient to have it imported directly (and consistently like in other places).